scripts/fix_any.py

```diff
@@ -301,17 +251,12 @@ def reorder(class_list):
     return class_list, forward_refs
 
 
-def fix_header(header, needs_conlist):
+def fix_header(header):
     """Fix imports and remove timestamp from header."""
     new_header = []
     for r in header:
-        # Any is needed for biblio so still need to import it
-        # r = re.sub(' Any, ', ' ', r) # noqa: E800
         if r.find(' timestamp: ') >= 0:
```

Comment:
@fsuits
There is `--disable-timestamp` option which is disabled generating this line.

Reply:
thank you @koxudaxi.  I tried adding this to the command in gen_oscal.py and the timestamp still appeared.  I do see it in the -h help listing for datamodel-codegen so it appears to be present in the version.  I may not be invoking it properly.

Reply:
The option works on my envriomment.
```bash
$ datamodel-codegen  --version                                                       
0.9.0
$ echo "{}"| datamodel-codegen --input-file-type jsonschema --disable-timestamp 
# generated by datamodel-codegen:
#   filename:  <stdin>

from __future__ import annotations
...

```
But, let's ignore my point out. I will check it when I know how to generete `compliance-trestle` models in locally.  Thank you.